README.md

```diff
@@ -95,13 +95,11 @@ As much as possible, this repo tries to follow the single-file implementation ru
 <!-- start citing -->
 
```

Comment:
I would add a sentence: "If you use MORL Baselines in your research, please cite our NeurIPS 2023 paper:"